src/main/Application.h

```diff
@@ -221,6 +222,7 @@ class Application
     virtual WorkScheduler& getWorkScheduler() = 0;
     virtual BanManager& getBanManager() = 0;
     virtual StatusManager& getStatusManager() = 0;
+    virtual SurveyDataManager& getSurveyDataManager() = 0;
```

Comment:
We need to bump OVERLAY_PROTOCOL_VERSION to indicate a version of core that supports new-style survey.

Reply:
Done in a10862017a818e19da975d0941a0dd05af4b77de

Reply:
Note: if this change lands in v21.1.0, we will need to bump overlay version again from 33 to 34, as we expect to bump it to 33 in v21.0.0.

Reply:
Bumped in 0859d2747e465743780b4e32071e98ac37766f49